### ISSUE TITLE
Security Issue: minimist

### ISSUE THREAD

**Issue by rob4629** [DevExpress/testcafe-browser-provider-browserstack#113]
`npm audit` shows 1 vulnerability in the `testcafe-browser-provider-browserstack` library:

```sh
                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.2.1 <1.0.0 || >=1.2.3                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ testcafe-browser-provider-browserstack [dev]                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ testcafe-browser-provider-browserstack > jimp > @jimp/custom │
│               │ > @jimp/core > mkdirp > minimist                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1179                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 5122 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by LavrovArtem** [comment]
Thank you for the research. We'll fix this problem as soon as possible.

**Comment by AndreyBelym** [comment]
Closed by #117.
